acl_loader/main.py

```diff
@@ -114,8 +114,13 @@ def __init__(self):
         self.tables_db_info = {}
         self.rules_db_info = {}
         self.rules_info = {}
-        # Load global db config. This call is no-op in single npu platforms
-        SonicDBConfig.load_sonic_global_db_config()
+        num_asic = multi_asic.get_num_asics()
+        if num_asic > 1:
+            # Load global db config
+            SonicDBConfig.load_sonic_global_db_config()
+        else:
+            SonicDBConfig.initialize()
```

Comment:
Need not call SonicDBConfig.initialize() explicitly in case of single ASIC platform, as the SonicDBConfig will load the default config automatically. If we load again it will throw an error ( this was the original behavior as well) (Ref: https://github.com/Azure/sonic-swss-common/blob/7afd43d1bf75c95dd1db86a4562d483c0da538fa/common/dbconnector.cpp#L176)

Reply:
It does not harm to explicitly initialize for single ASIC, and I think this is also a better practice than depending on implicit initialization in later function calls.

Reply:
Sure Qi, Agreed !